doc/source/index.rst

```diff
@@ -53,41 +63,24 @@ Ray comes with libraries that accelerate deep learning and reinforcement learnin
    :caption: Installation
 
    installation.rst
-   deploy-on-kubernetes.rst
-   install-on-docker.rst
-   installation-troubleshooting.rst
 
 .. toctree::
    :maxdepth: 1
-   :caption: Getting Started
+   :caption: Tutorial
 
    tutorial.rst
-   api.rst
    actors.rst
-   using-ray-with-gpus.rst
-   signals.rst
-   async_api.rst
+   resources.rst
+   examples.rst
 
 .. toctree::
    :maxdepth: 1
-   :caption: Cluster Usage
+   :caption: Cluster Setup
 
+   deploy-on-kubernetes.rst
```

Comment:
not sure about ordering here - maybe autoscaler instructions should go before kubernetes until it's better supported

Reply:
yep, got it.